common/config/rush/pnpm-lock.yaml

```diff
@@ -8692,7 +8696,6 @@ packages:
     dependencies:
       aws-cdk-lib: 2.29.1_constructs@10.1.25
       constructs: 10.1.25
-    dev: true
```

Comment:
Why did this get deleted?

Reply:
The `cfn-nag`package in `swb-reference` is in dependencies, so it is no longer a devDependencies only package